tanner/emulators/sqli.py

```diff
@@ -46,8 +48,10 @@ def map_query(self, attack_value):
                 error_result = 'You have an error in your SQL syntax; check the manual\
                                 that corresponds to your MySQL server version for the\
                                 right syntax to use near {} at line 1'.format(attack_value['id'])
+                self.logger.exception('Error while executing MySQL %s', error_result)
             else:
                 error_result = 'SQL ERROR: near {}: syntax error'.format(attack_value['id'])
+                self.logger.exception('Error while executing SQL %s', error_result)
```

Comment:
same here, we should log exception only in case it's tanner exception and not something we want emulate

Reply:
Okay